src/index.js

```diff
@@ -6,22 +6,103 @@ import Manager from './Manager';
 import Room from './Room';
 import Hotel from './Hotel';
 
+// let users;
+// let rooms;
 
+// var today = new Date();
+// var date = today.getFullYear()+'/'+(today.getMonth()+1)+'/'+0+today.getDate();
 
+// function getData(type) {
+// 	const root = 'https://fe-apps.herokuapp.com/api/v1/overlook/1904';
+// 	const url = `${root}${type}`;
+// 	const promise = fetch(url)
+// 	                .then(data => data.json());
+// 	return promise;
+// }
+
+// getData('/users/users').then(function(user) {
+//   	users = user.users;
+// 	  users.forEach(function(user) {
+// 	})
+
+// getData('/bookings/bookings').then(function(booked) {
+// 		bookings = booked.bookings;
+// 		const room = new Room(bookings);
+//   	// console.log(users)
+
+// getData('/rooms/rooms').then(function(room) {
+// 	  rooms = room.rooms;
+// 		const hotel = new Hotel(rooms);
+// 		console.log(rooms)
+// 		$('.room_available-text').text('%' + hotel.percentBooked(bookings, rooms, date));
+	  
+// 	})
+
+// })
+
+var users;
+var rooms;
+var bookings;
+var userId;
+
+
+var today = new Date();
+var date = today.getFullYear()+'/0'+ (today.getMonth()+1)+'/'+'0'+ today.getDate();
+
+function getData(type) {
```

Comment:
I like how you've made a generic fetch call that you can use for your api request, and you tailor the .then() callbacks functions for each data type. 

One thing to consider is how you'll be testing these functions.  You may consider breaking the getData function into its own file, perhaps of api calls you can export into other modules. Then maybe each data type class can have its own async method that would call that api function to do the class-specific logic?